### PR TITLE
Refactor arguments

### PR DESCRIPTION
**Pull request by ErinCain**
I exposed all betas following naming convention established for `fallRunDSM`. I also added tests for all refactored equations to check that I did not change any outputs in the refactor.

Note: There are still some todos describing sources for documentation (I am compiling a list and will address all these)

### CODE REVIEW

File: R/survival.R [hunk 1]
```diff
@@ -1,89 +1,75 @@
...
+                          .avg_temp_thresh = -0.717,
+                          .high_predation = -0.122,
+                          ..surv_juv_rear_contact_points =  -0.00368,
+                          ..surv_juv_rear_prop_diversions = -0.351,
+                          ..surv_juv_rear_total_diversions = -0.00063,
+                          .stranded = -1.939,
+                          .medium = 1.48,
+                          .large = 2.223,
+                          .floodplain = 0.47,
+                          .surival_adjustments = 1){ # TODO figure out how we want to add in scenario surv_adjustments
```
Comment (ergz) [inline R/survival.R:37]: add `.surival_adjustments` to param list in docs, just as a place holder for now

**Review by ergz — COMMENTED** [review]
just a small addition in the docs, but also when I build the package I get the following during the documentation build 

```
winterRunDSM/man/surv_juv_rear.Rd:55: unknown macro '\cpde'
```

looks like a typo for that documentation line

File: R/survival.R (hunk 1 above)
Comment (ErinCain) [reply R/survival.R:37]: Added

**Comment by ErinCain** [comment]
> just a small addition in the docs, but also when I build the package I get the following during the documentation build
> 
> ```
> winterRunDSM/man/surv_juv_rear.Rd:55: unknown macro '\cpde'
> ```
> 
> looks like a typo for that documentation line

Fixed typo

**Pull request merged**